### PR TITLE
add working menu

### PR DESCRIPTION
**Pull request by paulinagorecka**
Add working menu :smile_cat:

### CODE REVIEW

File: src/js/app.js [hunk 1]
```diff
@@ -7,15 +7,58 @@
...
+/*
 var main = new UI.Card({
-  title: 'Pebble.js',
+  title: 'Bicing.js',
   icon: 'images/menu_icon.png',
-  subtitle: 'Hello World!',
+  subtitle: 'Hello! Wanna bike?',
   body: 'Press any button.'
-});
+});
```
Comment (martinciu) [inline src/js/app.js]: maybe remove this comment?

File: src/js/app.js [hunk 2]
```diff
@@ -7,15 +7,58 @@
...
+})
 
 main.show();
 
+myCards = [
+  new UI.Card({
+    title: 'GeoCARD'
+  }),
+
+  new UI.Card({
```
Comment (martinciu) [inline src/js/app.js]: remove empty line

File: src/js/app.js [hunk 3]
```diff
@@ -7,15 +7,58 @@
...
+    title: 'AjaxCARD'
+  }),
+
```
Comment (martinciu) [inline src/js/app.js]: remove empty line

**Comment by paulinagorecka** [comment]
Done.

**Comment by martinciu** [comment]
:+1:

**Pull request merged**